components/Player/PlayerTabs.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import TabBar from '../TabBar';
 import { playerPages } from '../Header/Pages';
 
-const playerPagesMapped = (accountId) => playerPages.map((e) => Object.assign({}, e, {
-  route: `/players/${accountId}/${e.name.toLowerCase()}`,
-  label: e.name,
+const playerPagesMapped = (accountId) => playerPages.map(page => ({
+  route: `/players/${accountId}/${page.name.toLowerCase()}`,
```

Comment:
I would comment that if we are making design/code style decisions because the IDE isn't working, it probably signals we shouldn't be using those IDE's instead of changing our code.